### PR TITLE
Week 2 Problem Set - Ohm's Law

### PR DESCRIPTION
**Pull request by splazatorres**
Please review this week2 branch for mistakes, lack of code, etc. and give comments for improvement so it can be merged into master.

### CODE REVIEW

**Comment by splazatorres** [comment]
Binder link for reviewers:  <a href='https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-splazatorres/binder-R?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-splazatorres%26branch%3Dweek2%26urlpath%3Drstudio'></a>

File: week2.Rmd [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+After doing this lesson you should be able to:
+
+1. Define what each variable of the Ohm's Law equation stands for.
+2. Create a tibble dataframe, use the dataframe for creating plots with ggplot2, and use the dataframe to perform calculations for variables in the Ohm's Law equation.
+3. Visualize Ohm's Law graphically.
+
+## Background
+
+Ohm's Law is a concept that focuses on the study of electrical circuits. The equation for Ohm's Law is:
```
Comment (KevRozmiarek) [inline week2.Rmd]: You should expand on what this "concept" is.

File: week2.Rmd [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## Questions
+
+1. In Ohm's Law $R$ represents the _____________ (fill in the blank) in Ohms.
+
+2. Rearrange the Ohm's Law equation to solve for $I$. What would this equation imply? Explain the variables in the equation. 
+
+3. What is the kind of relationship between $V$ and $I$ graphically for Ohm's Law? What does the slope represent?
+
+4. Apply your dataframe and ggplot2 knowledge. Create a dataframe using this rearranged version of Omh's Law $V=IR$. The dataframe's $R$ variable should start at 1 and end at 6, incrementing by 1 from one number to another. The $I$ variable should go from 0 to 1000 incrementing by 200. Calculate $V$ for every row in your dataframe using the equation above. Add the results of $V$ as a new column in your dataframe. Use your new $V$ values alongside the $I$ values to make a plot showing the relationship $R=V/I$.
```
Comment (KevRozmiarek) [inline week2.Rmd]: The detail is good here!

**Review by KevRozmiarek — COMMENTED** [review]
The example section is fantastic. The background section needs to be expanded upon with more conceptual talk of what resistance or impedance is.

**Pull request merged**